core/src/main/java/kafka/server/builders/KafkaApisBuilder.java

```diff
@@ -166,11 +164,6 @@ public KafkaApisBuilder setTime(Time time) {
         return this;
     }
 
-    public KafkaApisBuilder setTokenManager(DelegationTokenManager tokenManager) {
```

Comment:
Why are we removing this?

Reply:
The `KafkaApisBuilder` is only used in `KRaftMetadataRequestBenchmark` and the `tokenManager` is always `null`, so I think we can safely remove it.

Reply:
Yes they are currently unused but I think the goal was to start using them over the direct constructors. It seems we've not really started using them yet but I think we should keep these methods.

Reply:
Added related function back. Thank you for your explanation.